.github/workflows/deploy-on-pr-merge.yaml

```diff
@@ -3,10 +3,17 @@ name: deploy-on-pr-merge
 on:
   push:
     branches:
-      - master
+      - master # snapshot deployment
+      - 13.x
+  pull_request:
+    types:
+      - closed # pr-labelled deployment (only if PR is merged to master or 13.x)
 
 jobs:
   deploy-snapshot:
+    name: deploy snapshot/PR-labelled version
+    if: github.event_name == 'push' || (github.event.pull_request.merged == true && (github.event.pull_request.base.ref == 'master' || github.event.pull_request.base.ref == '13.x'))
```

Comment:
I would try condensing
```(github.event.pull_request.base.ref == 'master' || github.event.pull_request.base.ref == '13.x')```
into
```contains(fromJson('["master", "13.x"]'), github.event.pull_request.base.ref)```

This should help adding more branches to this workflow.

Never tried it before, so no guarantees:-)